### PR TITLE
Update changelog with UI notes

### PR DESCRIPTION
**Pull request by dvonanderson**
- Update changelog with UI notes to the New and Improved section.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -35,6 +35,8 @@ Canonical reference for changes, improvements, and bugfixes for Boundary.
     relative to the last result received.
   * config: add new controller max_page_size field for controlling the default and max size
     of pages when paginating through results.
+* ui: Add full UI support for Storage Policies manageing the lifecycle of Session Recordings.
```
Comment (johanbrandhorst) [inline CHANGELOG.md:38]: Maybe this should be a subsection under `Policies` above?
```suggestion
* ui: Add full UI support for Storage Policies managing the lifecycle of Session Recordings.
```

**Review by johanbrandhorst — CHANGES_REQUESTED** [review]

**Review by johanbrandhorst — COMMENTED** [review]
Did you see my note about adding this as a sub-heading to the `Policies` section?

**Comment by dvonanderson** [comment]
> Did you see my note about adding this as a sub-heading to the `Policies` section?

@johanbrandhorst that is a great point.  Will push up in a sec

File: CHANGELOG.md [hunk 2]
```diff
@@ -35,7 +37,7 @@ Canonical reference for changes, improvements, and bugfixes for Boundary.
     relative to the last result received.
   * config: add new controller max_page_size field for controlling the default and max size
     of pages when paginating through results.
-
+
```
Comment (johanbrandhorst) [inline CHANGELOG.md:40]: 😬 mind removing the extra tab?

**Review by johanbrandhorst — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (dvonanderson) [reply CHANGELOG.md:40]: haha got ya

Comment (johanbrandhorst) [reply CHANGELOG.md:39]: I think you accidentally removed the wrong tab?

Comment (dvonanderson) [reply CHANGELOG.md:40]: sorry about that @johanbrandhorst my editor insist on making that URL markdown information stay on the same line.  Let me know if that is what you were referring to.

Comment (dvonanderson) [reply CHANGELOG.md:40]: Ahh see it now

**Review by ZedLi — APPROVED** [review]

**Review by johanbrandhorst — APPROVED** [review]

**Pull request merged**